packages/js/app/src/views/SignupScreens/Signup2.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import {TextInput, Text, View, StyleSheet, ImageBackground, TouchableOpacity} from 'react-native';
+import {NativeStackScreenProps} from "@react-navigation/native-stack";
+import Header from '../../components/Header';
+import TextInputBox from "../../components/TextInputBox";
+import colors from '../../constants/Colors';
+
+type RootStackParamList = {
+    Login: undefined,
+    Signup1: undefined,
+    Signup2: undefined,
+    Splash: undefined,
+};
+
+type Props = NativeStackScreenProps<RootStackParamList, 'Signup2'>;
+
+const styles = StyleSheet.create({
+    container: {
+        alignItems: 'center',
+        backgroundColor: colors.white,
+        width: '100%',
+        height: '100%',
+    },
+    backgroundImage:{
+        width: '100%',
+        height: '100%',
+        alignItems: 'center',
+    },titleContainer:{
+        borderWidth: 1,
+        top: '12%',
+        flex: 1.75,
+        justifyContent: 'center',
+    }, signupContainer:{
+        borderWidth: 1,
+        flex: 2.5,
+        justifyContent: 'center',
+    },signupButtonContainer:{
```

Comment:
go to packages/js and run 'yarn prettier:fix'. Will get rid of styling issues like these